iocore/hostdb/P_RefCountCacheSerializer.h

```diff
@@ -157,7 +159,7 @@ RefCountCacheSerializer<C>::write_partition(int /* event */, Event *e)
   // for item in this->partitionItems
   // write to disk with headers per item
 
-  for (unsigned int i = 0; i < this->partition_items.length(); i++) {
+  for (unsigned int i = 0; i < this->partition_items.size(); i++) {
```

Comment:
this can change from:

```c++
  for (unsigned int i = 0; i < this->partition_items.size(); i++) {
    RefCountCacheHashEntry *entry = this->partition_items[i];
```

to

```c++
for (auto const& entry : this->partition_items) {
```